### ISSUE TITLE
Markup usability

### ISSUE THREAD

**Issue by andrewgbuck** [AppGeo/GPV#124]
The Markup workflow and lifecycle is not obvious.

**Comment by MikeOlkinSwsc** [comment]
I disagree. I find it to be much more intuitive in 5.1, with everything in one place, with the clear distinction between quick sketches & markup groups.
